src/Reflection/ReflectionClass.php

```diff
@@ -177,7 +176,7 @@ public function __toString()
      * @param string $className
      * @return ReflectionClass
      */
-    public static function createFromName($className)
+    public static function createFromName(string $className)
```

Comment:
Can't - `ReflectionObject` extends this, and ` : static` isn't a valid return type :/